webrtc.html

```diff
@@ -9712,7 +9712,9 @@ <h2>Attributes</h2>
             <dt><dfn data-idl><code>canInsertDTMF</code></dfn> of type <span class=
             "idlAttrType"><a>boolean</a></span>, readonly</dt>
             <dd>
-              <p>Whether the <a>RTCDTMFSender</a> is capable of sending DTMF.</p>
+              <p>Whether the <a>RTCDTMFSender</a> <var>dtmfSender</var> is capable of
+              sending DTMF. On getting, the user agent MUST <a>update the [[\CanInsertDTMF]]</a>
+              internal slot of <var>dtmfSender</var> and return it.</p>
```

Comment:
What @stefhak said. Since `insertDTMF` doesn't *update* the [[CanInsertDTMF]] slot, it would keep failing until the user tries to get the attribute in JS, so this won't work.

As I mentioned in https://github.com/w3c/webrtc-pc/pull/1957#discussion_r208940444 I think the point of internal slots would be to move the `true`/`false` determination away from the getter, to where the changes may happen.